samples/bluetooth/direction_finding_connectionless_rx/src/main.c

```diff
@@ -65,6 +70,11 @@ static struct bt_le_scan_cb scan_callbacks = {
 	.recv = scan_recv,
 };
 
+static inline uint32_t adv_interval_to_ms(uint16_t interval)
+{
+	return interval * 5 / 4;
+}
```

Comment:
I'd just avoid the `inline` qualifier here. The compiler should inline it since it is only used once. 

```suggestion
static uint32_t adv_interval_to_ms(uint16_t interval)
{
	return interval * 5 / 4;
}
```

I see that some other samples just use a macro. 
Perhaps macros in gap.h to transform these sort of values should be added, as we do this conversion quite a lot.

Reply:
Inline removed. Function changed to `sync_create_timeout_get`. I've added a macro `BT_GAP_ADV_INTERVAL_TO_MS` in `gap.h`